### DISCUSSION TITLE
[Process] Access to slack channel

### DISCUSSION THREAD

**Discussion by tangerine-orange** [maplibre/maplibre-gl-js#2816] [General]
Hi all. In the contribution guide for the project, it mentions joining the [Slack Channel](https://osmus.slack.com/archives/C01G3D28DAB) to discuss proposed changes before submitting a PR. I tried to join the Slack Channel, but am unable to because I don't have an @openstreetmap.us email address. Is there a protocol for getting an invitation to the channel?

**Comment by HarelM** [comment]
I don't have a openstreetmap.us email and I'm on that channel.
I'm not sure you need anything special besides a slack account...
